### ISSUE TITLE
`theme_guide(key.width, key.height)` do not work as expected

### ISSUE THREAD

**Issue by teunbrand** [teunbrand/legendry#41]
Sorry to bother you again. 
is there something problem with the height and width adjustment of the colorbar in the `theme_guide` function provided by `legendry`?
Using `key.width` and `key.height` seems to have no effect. But it is valid to use `theme` instead of `theme_guide` .

See details:
[stackoverflow-question](https://stackoverflow.com/questions/79289871/legendry-and-ggplot-theme-guide-adjustment-of-height-and-width-failure/79289898#79289898)

_Originally posted by @Breeze-Hu in https://github.com/teunbrand/legendry/issues/37#issuecomment-2550257636_

**Comment by Breeze-Hu** [comment]
I am sorry about the possibility of interrupting your vacation.

Could you please tell me if there is a possibility that the parameters such as `position = 'inside'` or `theme = theme(legend.position = 'inside)` in `guide_colbar` could be adjusted in the future to be similar to those similar to `guide_colorbar` in ggplot2. 
Right now it doesn't seem to be able to put the colorbar into the panel, and I'm not quite sure if it's the same due to this bug.

**Comment by teunbrand** [comment]
Yes it should work similar to `guide_colourbar()` and will need to be adjusted. This is different than he key width/height, so I've opened a separate issue in #42.

**Comment by Breeze-Hu** [comment]
Thank you very much! Looking forward to your subsequent great works! : )

**Comment by Breeze-Hu** [comment]
Great！
